.github/workflows/integ-tests.yml

```diff
@@ -0,0 +1,62 @@
+name: Integration Tests
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+env:
+  AWS_REGION : "us-west-2"
+  REGION : "us-west-2"
+  AMZN_BRAKET_OUT_S3_BUCKET : ${{ secrets.bucket_name }}
+  REPOSITORY_NAME : ${{ secrets.repository_name }}
+  ACCOUNT_ID : ${{ secrets.account_id }}
+  BUILD_RESULTS_PATH : "build_results.txt"
+
+permissions:
+  id-token: write   # This is required for requesting the JWT
+  contents: read    # This is required for actions/checkout
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        python-version: ["3.10"]
+        container-type: ["base", "pytorch", "tensorflow"]
+    runs-on: ${{ matrix.os }}
+    env:
+      PREBUILD_TAG : ${{ matrix.container-type }}_prebuild_tag
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v5
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: configure aws credentials
+      uses: aws-actions/configure-aws-credentials@v4
+      with:
+        role-to-assume: arn:aws:iam::${{ env.ACCOUNT_ID }}:role/GitHubIntegTestBuildRole
+        role-session-name: github_role_session
+        aws-region: ${{ env.AWS_REGION }}
+    - name: Install Build dependencies
+      run: |
+        pip install --upgrade pip
+        pip install -r src/requirements.txt
+    - name: ECR Login
+      run: |
+        aws ecr get-login-password --region ${{ env.AWS_REGION }} | docker login --username AWS --password-stdin ${{ env.ACCOUNT_ID }}.dkr.ecr.${{ env.AWS_REGION }}.amazonaws.com
+        aws ecr get-login-password --region us-east-1 | docker login --username AWS --password-stdin 763104351884.dkr.ecr.us-east-1.amazonaws.com
```

Comment:
Does this need to be an environment variable?

Reply:
No, that's the public account ID for SageMaker DLC. It's also visible in our Dockerfiles.